manifests/4.9/supported-nic-ids_v1_configmap.yaml

```diff
@@ -22,7 +22,6 @@ data:
   Nvidia_mlx5_ConnectX-5_Ex: 15b3 1019 101a
   Nvidia_mlx5_ConnectX-6: 15b3 101b 101c
   Nvidia_mlx5_ConnectX-6_Dx: 15b3 101d 101e
-  Nvidia_mlx5_ConnectX-6_Lx: 15b3 101f 101e
```

Comment:
is there a reason why we remove this device?

Reply:
Same as here: https://github.com/openshift/sriov-network-operator/pull/639#discussion_r833046689